src/Skia/Avalonia.Skia/SkiaOptions.cs

```diff
@@ -16,6 +16,6 @@ public class SkiaOptions
         /// <summary>
         /// The maximum number of bytes for video memory to store textures and resources.
         /// </summary>
-        public long? MaxGpuResourceSizeBytes { get; set; }
+        public long? MaxGpuResourceSizeBytes { get; set; } = 1024 * 600 * 4 * 12; // ~28mb 12x 1024 x 600 textures. 
```

Comment:
This property is nullable with the default value I assume meaning "use the default size". Does it make sense to initialize the property here in that case?

Reply:
currently `null` means use skia default... and the value its set to here is use avalonia default.

Reply:
added comments.